website/components/Hero.module.css

```
@@ -0,0 +1,102 @@
+  opacity: 0;
+  transform: translateY(40px);
+  animation: slideIn 900ms cubic-bezier(0.2, 0.8, 0.2, 1) forwards;
+}
+.word:nth-child(2) {
+  animation-delay: 120ms;
+}
+.accent {
+  color: var(--gold);
+}
+
+@keyframes slideIn {
```

Stylelint (17.7.0)

[error] Expected keyframe name "slideIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)